tasks/e2e-kitchensink.sh

```diff
@@ -85,7 +85,7 @@ publishToLocalRegistry
 
 # Install the app in a temporary location
 cd $temp_app_path
-npx create-react-app test-kitchensink --template=file:"$root_path"/packages/react-scripts/fixtures/kitchensink
+npx create-react-app test-kitchensink --scripts-version=@skyscanner/backpack-react-scripts --template=file:"$root_path"/packages/react-scripts/fixtures/kitchensink
```

Comment:
Revert all changes in this file, and keep it consistent with CRA5,
Tao Fang is working on how to build and test CRA5